src/main/kotlin/org/bundleproject/json/github/Asset.kt

```diff
@@ -0,0 +1,17 @@
+package org.bundleproject.json.github
+
+data class Asset(
```

Comment:
Do we use every single piece of data passed by this object? If not only specify what you need